sagemaker-training-compiler/tensorflow/single_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p>This example notebook requires <strong>SageMaker Python SDK v2.92.0 or later.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>